ignite/pkg/protoanalysis/protoutil/utils.go

```diff
@@ -0,0 +1,281 @@
+// Utils holds some functions that mostly wrap cursor and make adding things easier.
```

Comment:
We try not to have any `utils` files in the repo.  Can you reorganize or group these functions in some other way.  If not, can you rename to `helpers.go`?

Is it necessary to export any of these functions?

Reply:
> Is it necessary to export any of these functions?

Yup, there's common stuff that many other use (like looking up a message, a service etc). I've renamed to helpers instead.